src/clj/server/csv.clj

```diff
@@ -0,0 +1,37 @@
+(ns server.csv
+  (:require
+   [clojure.string :as string]
+   [shoreleave.middleware.rpc :refer [defremote]]))
+
+(defremote csv-files
```

Comment:
@wmedrano What's the plan? Store CSVs on the BeagleBone and allow the client to download them? 
The dataflow I'm imagining in my head is this:

1. Poll data from MachineKit (or generate random data)
2. The client graphs/does stuff with that data
3. The client periodically will backup that data on the server. 
4. The user can download a backup if they want

I think it makes sense to have one remote call for backing up data. Something that takes a file name and a sequence of data.